app/src/main/res/drawable/emergency_home_fill0_wght400_grad0_opsz48.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The icon in lemmy-ui is 

![Screenshot_2023-08-29-10-52-38-142_mark via gp](https://github.com/dessalines/jerboa/assets/930215/9f2b5b13-2aa5-4ab3-9d62-b851a3674b94)

Which should be available as https://fonts.google.com/icons?selected=Material+Icons+Outlined:report_problem:&icon.set=Material+Icons&icon.query=warning or Warning Amber.

I don't know if there's a universal spoiler icon that ppl have settled on, but it should probably match lemmy-ui at least for now.

Reply:
Everything else looks good.

Reply:
The icon I used was the one Reddit and other client used for Spoilers

Reply:
mmk then, I'll make an issue for lemmy-ui to add this one.

Reply:
Well I am not against changing icon, especially if Lemmy also supports that kind of spoiler that the icon was used for